it/google-cloud-platform/src/test/java/com/google/cloud/teleport/it/gcp/launcher/PipelineOperatorTest.java

```diff
@@ -102,7 +102,7 @@ public void testWaitUntilDoneTimeout() throws IOException {
     when(client.getJobStatus(any(), any(), any())).thenReturn(JobState.RUNNING);
     Result result = new PipelineOperator(client).waitUntilDone(DEFAULT_CONFIG);
     assertThat(result).isEqualTo(Result.TIMEOUT);
-    assertThatResult(result).hasTimedOut();
+    TemplateAsserts.assertThatResult(result).hasTimedOut();
```

Comment:
Same, static imports look cleaner / idiomatic

Reply:
Same, not intended. Fixed.